### PR TITLE
Fix bios boot python script due to 1.8.x changes

### LINKED ISSUE #8261: bios-boot-tutorial.py incompatible with current system contracts
When running bios-boot-tutorial.py with current system contracts, I get the error

```
Reading WASM from /opt/src/eosio.contracts/build/contracts/eosio.system/eosio.system.wasm...
Publishing contract...
Error 3070000: WASM Exception
Error Details:
env.is_feature_activated unresolveable
```

The script needs to activate `PREACTIVATE_FEATURE` before setting the contract.

### PR DESCRIPTION
**Pull request by iamveritas**
### Change Description

1. Adds `PREACTIVATE_FEATURE` before eosio.system.
2. Adds the rest of the features that could be activated if user wants to.
3. Update README.md file.

Resolves #8261 

- [x] Documentation Updates

### CODE REVIEW

File: tutorials/bios-boot-tutorial/bios-boot-tutorial.py [hunk 1]
```diff
@@ -293,9 +296,53 @@ def stepCreateTokens():
...
+    # sleep(1)
+    # # NO_DUPLICATE_DEFERRED_ID
+    # retry(args.cleos + 'push action eosio activate \'["4a90c00d55454dc5b059055ca213579c6ea856967712a56017487886a4d4cc0f"]\' -p eosio')
+    # sleep(1)
+    # # ONLY_LINK_TO_EXISTING_PERMISSION
+    # retry(args.cleos + 'push action eosio activate \'["1a99a59d87e06e09ec5b028a9cbb7749b4a5ad8819004365d02dc4379a8b7241"]\' -p eosio')
+    # sleep(1)
+    # # RAM_RESTRICTIONS
+    # retry(args.cleos + 'push action eosio activate \'["4e7bf348da00a945489b2a681749eb56f5de00b900014e137ddae39f48f69d67"]\' -p eosio')
+    # sleep(1)
```
Comment (heifner) [inline tutorials/bios-boot-tutorial/bios-boot-tutorial.py]: Seems by default we should enable all of these.

Comment (iamveritas) [reply tutorials/bios-boot-tutorial/bios-boot-tutorial.py]: I do not understand why 'by default' it should enable all of them, can you please explain more?
The way I see it, we leave it like that and the user is choosing which one to uncomment based on which one she wants to experiment with.

Comment (heifner) [reply tutorials/bios-boot-tutorial/bios-boot-tutorial.py]: I guess I see the bios boot script as setting up minimum current chain. We expect/desire all these features to be enabled on eosio chains. They are simply features that required hard-fork to enable. If we started a new chain we would start it with all these enabled.

Comment (iamveritas) [reply tutorials/bios-boot-tutorial/bios-boot-tutorial.py]: ah, ok, I did not get it that way, that we expect/desire all these features to be enabled on eosio chains. alrighty then, I will uncomment and have them enabled by default. thank you @heifner 👍

File: tutorials/bios-boot-tutorial/bios-boot-tutorial.py [hunk 2]
```diff
@@ -293,9 +296,53 @@ def stepCreateTokens():
     run(args.cleos + 'push action eosio.token issue \'["eosio", "%s", "memo"]\' -p eosio' % intToCurrency(totalAllocation))
     sleep(1)
 def stepSetSystemContract():
+    # Nearly all of the protocol upgrade features introduced in v1.8 first require a special protocol
```
Comment (heifner) [inline tutorials/bios-boot-tutorial/bios-boot-tutorial.py]: It is not "Nearly all" it is "all".

File: tutorials/bios-boot-tutorial/bios-boot-tutorial.py [hunk 3]
```diff
@@ -293,9 +296,53 @@ def stepCreateTokens():
...
+    sleep(5)
+
+    # install eosio.system
     retry(args.cleos + 'set contract eosio ' + args.contracts_dir + '/eosio.system/')
     sleep(1)
+
+    # activate remaining features
+    # GET_SENDER
+    retry(args.cleos + 'push action eosio activate \'["f0af56d2c5a48d60a4a5b5c903edfb7db3a736a94ed589d0b797df33ff9d3e1d"]\' -p eosio')
+    sleep(1)
```
Comment (heifner) [inline tutorials/bios-boot-tutorial/bios-boot-tutorial.py]: I don't think any of these sleeps between `retry` are needed.

File: tutorials/bios-boot-tutorial/bios-boot-tutorial.py (hunk 2 above)
Comment (iamveritas) [reply tutorials/bios-boot-tutorial/bios-boot-tutorial.py]: "nearly all" is used here "https://developers.eos.io/eosio-nodeos/docs/consensus-protocol-upgrade-process"
I will correct.

**Comment by iamveritas** [comment]
let's first merge this one https://github.com/EOSIO/eos/pull/8297 and then move to 2.x and development

**Pull request closed without merging**